Jenkinsfile.integration

```diff
@@ -3,7 +3,20 @@
 
 def compute_node='storage-rookcheck'
 // the prefix we want to use for the rookcheck environment
-def cluster_prefix="${currentBuild.fullProjectName.replace('/', '_')}-${currentBuild.number}_"
+// def cluster_prefix="${currentBuild.fullProjectName.replace('/', '_')}-${currentBuild.number}_"
+//
+// We need to set cluster_prefix as short as possible, there are two reasons
+// for this:
+// 1) too long node names constructed from cluster_prefix make kubectl annotate
+//    fails when the network isn't set up and the name is long. This is possibly
+//    an upstream kubernetes bug that we need to investigate.
+// 2) it is not super usable to operate with long names. For example, when using
+//    openstack client to list any kind of resources in terminal like servers,
+//    volumes, etc. the output tables turn into barely readable ascii soup.
+
+def branch = params.BRANCH_NAME.toLowerCase()
```

Comment:
is `BRANCH_NAME` is not always available (eg. not in the periodic jobs like http://storage-ci.ses.suse.de:8080/view/rookcheck/job/rookcheck-devel_storage_7-periodic/ )